src/main/java/frc/robot/Constants.java

```diff
@@ -144,25 +149,21 @@ public static final class ModuleConstants {
     public static final double kDrivingMaxOutput = 1.0;
 
     public static final PIDGains kTurningPIDGains =
-        new PIDGains(1, 0, 0); // TODO: tune values for the turning motor
-    public static final double kTurningFF = 0.0;
+        new PIDGains(4, 0, 0.075); // TODO: tune values for the turning motor
+    public static final double kTurningFF = 0.0; // TODO: tune values for Feed Forward
```

Comment:
Can this Todo be resolved?  If not, please add more to explain when this should be resolved

Reply:
currently can not be resolved without a robot